### DISCUSSION TITLE
Google Ads conversion code

### DISCUSSION THREAD

**Discussion by jlores** [PrestaShop/PrestaShop#36833] [Q&A]
Hello,

I need to add conversion tracking code in Prestashop 8.1, but I don't know where to add the purchase event code. 

Could someone guide me?

Best regards

**Comment by kpodemski** [comment]
Hello @jlores 

The most convenient method to integrate conversion tags is Google Tag Manager, and then integrating Google Tag Manager inside your PrestaShop. There are plenty of modules ready to help you with that: https://addons.prestashop.com/en/search?search_query=tag%20manager

**Comment by Shriharsh-Deshmukh** [comment]
Hello **@jlores**  
To add conversion tracking in PrestaShop 8.1:

1. **Open the Order Confirmation Template**:
   - Go to `themes/<your-theme>/templates/checkout/order-confirmation.tpl`.

2. **Add Your Tracking Code**:
   - Insert your tracking code at the end of the file, right before the closing `</body>` tag.

   Example:
   ```html
   <!-- Tracking code -->
   <script>
     // Example for Google Analytics
     gtag('event', 'purchase', {
       'transaction_id': '{$order.reference}',
       'value': '{$order.total_paid_tax_incl}',
       'currency': '{$currency.iso_code}'
     });
   </script>
   ```

3. **Save and Clear Cache**:
   - Save the changes and clear your PrestaShop cache from the back office.

This places the tracking code where it will run when an order is confirmed.

**Comment by jlores** [comment]
Hello,

Many thanks for your help, order-confirmation.tpl don't have </body> tag.
